src/main/java/org/beanpod/switchboard/controller/LogController.java

```diff
@@ -24,4 +31,15 @@
   public ResponseEntity<List<LogModel>> retrieveDeviceLogs(@PathVariable String serialNumber) {
     return ResponseEntity.ok(logDao.getDeviceLogs(serialNumber));
   }
+
+  @Override
+  public ResponseEntity<LogModel> createLog(@Valid CreateLogRequest createLogRequest) {
+    return Optional.of(createLogRequest)
+        .map(logMapper::toLogDto)
+        .map(logDao::createLog)
```

Comment:
Ideally, the request would go through `LogService` first.

Reply:
I have updated the code according to your suggestion.